python/tests/integration/test_methods.py

```diff
@@ -291,6 +291,7 @@ def test_search_looks_title_fields_filter(sdk: mtds.LookerSDK):
     assert "Order" in look.title
     assert look.description is None
 
+
 def test_search_look_and_run(sdk: mtds.LookerSDK):
```

Comment:
I don't see a test that exercises the new `__init__` generation support.

Reply:
there's actually no runtime behavior difference. All the models get an `__init__` as `@attrs` will auto-generate one for you unless you tell it not to. so then it becomes a "meta" level test of what does the written code look like and I added that to the codegen side

Reply:
haha, this change is because I had a local test failure due to missing data in my helltool. when I saved this file `black` reformatted it to add the blank line